hathor/p2p/states/hello.py

```diff
@@ -10,12 +12,15 @@
 if TYPE_CHECKING:
     from hathor.p2p.protocol import HathorProtocol  # noqa: F401
 
+logger = get_logger()
+
 settings = HathorSettings()
 
 
 class HelloState(BaseState):
     def __init__(self, protocol: 'HathorProtocol') -> None:
         super().__init__(protocol)
+        self.log = logger.new(remote=protocol.get_short_remote())
```

Comment:
Shouldn't it be enough on `BaseState`?

Reply:
I did it but it was logging as `BaseState` instead of each class. Maybe I did something wrong.